src/charts/styles/vscode.css

```diff
@@ -1,23 +1,72 @@
 :root {
-  --container-paddding: 20px;
+  --container-padding: 20px;
   --input-padding-vertical: 6px;
   --input-padding-horizontal: 4px;
   --input-margin-vertical: 4px;
   --input-margin-horizontal: 0;
 }
 
 body {
-  padding: 0 var(--container-paddding);
+  padding: 0 var(--container-padding);
   color: var(--vscode-foreground);
   font-size: var(--vscode-font-size);
   font-weight: var(--vscode-font-weight);
   font-family: var(--vscode-font-family);
   background-color: var(--vscode-editor-background);
 }
 
+/* inter-regular - latin-ext */
+@font-face {
+  font-family: "Inter";
+  font-style: normal;
+  font-weight: 400;
+  src: local(""), url("./inter-v3-latin-ext-regular.woff2") format("woff2"),
+    url("./inter-v3-latin-ext-regular.woff") format("woff");
+  font-display: swap;
+}
```

Comment:
Right, however Inconsolata is a monospaced font, Inter is not. I'm fine with using a different font, but should update the design to use a different font as well.

Reply:
What I mean is that we're now adding multiple `.woff(2)` files instead of a single variable font file, just like the [Inconsolata font file](https://github.com/autometrics-dev/vscode-autometrics/pull/31/files#diff-fc15058fd536bb0e82b86ac34f5e80f8632eebc4c35012756e767e7359778ece). Not really a blocker for now, but we should consider just adding a single variable font file.

Reply:
It's now a variable font...

Reply:
The variable font variant is bigger than the original files though, even now that I've optimized it so it doesn't include the italic styling